packages/cloud_firestore/cloud_firestore/darwin/Classes/FLTCloudFirestorePlugin.m

```diff
@@ -0,0 +1,774 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Is `darwin` a supported platform? Or is it a shared place to put common `ios`/`macos` code?

Reply:
Also, is this a mv of `packages/cloud_firestore/cloud_firestore/ios/Classes/FLTCloudFirestorePlugin.m`?

Reply:
Yes to both :)

Reply:
Wow! I had no idea of the `darwin` entrypoint, thanks for the info!